pkgs/build-support/agda/default.nix

```
@@ -45,7 +45,7 @@ let
     }:
     let
       libraryFile = mkLibraryFile pkgs;
-      pname = "agdaWithPackages";
+      pname = "${Agda.meta.mainProgram}" + "WithPackages";
```

Is this actually required? It can also be done with one string, rather than concatenation.

```suggestion
      pname = "${Agda.meta.mainProgram}WithPackages";
```